the_djangoist_clothes/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from uuid import uuid4
+
+class Garment(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    name = models.CharField(max_length=200)
+    description = models.TextField(blank=True)
```

Comment:
Again, feel free to experiment with different properties, here. It's fun, I promise! 😄